packages/screens/Mini/Network/ChangeNetworkScreen.tsx

```diff
@@ -73,14 +66,35 @@ export default function ChangeNetworkScreen({
                 paddingHorizontal: layout.spacing_x1_5,
               }}
               options={{
-                label: item?.label,
+                label: item?.displayName,
                 iconEnabled: false,
                 rightLabel: (
                   <View>
                     <Checkbox
-                      isChecked
-                      value={item?.label}
-                      onPress={() => {}}
+                      isChecked={selectedWallet?.networkId === item?.id}
+                      value={item?.displayName}
+                      onPress={() => {
+                        goBackTo();
```

Comment:
I think it'd better if we place goBackTo() at the end of other actions. This is kind of last action here. When replacing view, sometimes it stops all rendering and can cause some warning/errors because the code is trying to update inexist component.